tests/cli/app.py

```diff
@@ -121,6 +121,7 @@ def test_web_app_server(self):
         response = urlopen('http://localhost:11111')
         self.assertEqual(b"hello world", response.read())
         response.close()
+        sleep(.6)  # Test can fail in OSX without this wait.
```

Comment:
This looks like something is not working on Mac OS X and should be fixed. Mangling test to not catch the bug is not right to me.